Ginger/GingerCoreNET/GenAIServices/GenAIServiceHelper.cs

```diff
@@ -162,7 +171,23 @@ public async Task<string> StartNewChat(string chatBotRequest)
         }
 
 
+        private static string CredentialsCalculation(string value)
+        {
+            ValueExpression valueExpression = new();
```

Comment:
Consider using singleton pattern here, so vale expression object will be initialized once and reused afterwards